lib/TalkSession.php

```diff
@@ -92,43 +92,32 @@ protected function getValues(string $key): array {
 
 	protected function getValue(string $key, string $token): ?string {
 		$values = $this->getValues($key);
```

Comment:
Could you check how this behaves if you reopen the session before reading it again?

Reply:
this function is "protected" and the usages in this file that "write" it afterwards do it already

Reply:
But still produces arrays like:
```
│[error] 2022-11-02T17:56:20.000 spreed           join before["deorokdx"]1667404580.026                                                                                                  │
│[error] 2022-11-02T17:56:20.000 spreed           leave before["deorokdx"]1667404580.0406                                                                                                │
│[error] 2022-11-02T17:56:20.000 spreed           leave after[]1667404580.0694                                                                                                           │
│[error] 2022-11-02T17:56:20.000 spreed           join after["deorokdx","8bitgnry"]1667404580.084    
```